tests/Fusonic/Linq/Test/MiscTest.php

```diff
@@ -36,19 +32,25 @@ public function testConcat_ReturnsConcatenatedElements()
         $this->assertEquals("d", $all[3]);
     }
 
-    public function testConcat_ThrowsArgumentExceptionIfNoTraversableArgument()
+    /**
+     * @expectedException InvalidArgumentException
```

Comment:
Yeah sure, narrowing it down further with exception message can make sense ;)
There is now black and white here ... just different approaches with pros&cons

Thx for the PR by the way ;)